types/three/src/renderers/shaders/UniformsLib.d.ts

```diff
@@ -1,3 +1,7 @@
+export interface UniformBuilder<TValue> {
```

Comment:
This might be just be my unfamiliarity with WebGL, but when I see "UniformBuilder", I think of a builder function/class that returns a `Uniform`. Does it not work to use the name `Uniform` or am I missing something? Alternatively we could beef up `IUniform` to have a generic that defaults to `unknown`.

Reply:
I think thats a fair assumption, I wanted to avoid calling it Uniform because there is/was a class/function called Uniform in `three`. However, we could beef up IUniform like so?

```ts
interface IUniform<TValue = any> {
value: TValue
}
```

if you're happy with that? We wanna do a default instead of trying to type all the existing versions of IUniform

Reply:
Yeah, that looks good. Perhaps `unknown` would be a safer default?

Reply:
yeah, I'm not actually sure the difference between `unknown` and `any`. So I'll take your word and do `unknown`

Reply:
Looks like it fails too many tests, so we can just stick with `any` for now. I'll work on making it `unknown` in the f

Reply:
Cool changed 👌🏼